libbeat/docs/processors-using.asciidoc

```diff
@@ -847,6 +847,7 @@ processors:
     #match_source_index: 4
     #match_short_id: true
     #cleanup_timeout: 60
+    #labels.dedot: false
```

Comment:
Just wanted to comment that we so far use `dedot.enabled` but realised not even the naming of our config options are consistent :-(

Reply:
Oh, I see `dedot.enabled` in http module. We can think about converging these options in a follow up.
The only thing is that for k8s we will have to think if we want two options, one for labels and another one for annotations.

Reply:
Also not sure, both works for me.